app/templates/notesheet/layout.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="panel panel-default">
+  <div class="panel-heading js-sheets-select">
+    <input class="form-control" id="sheets-select" name="notesheet" placeholder="Select music sheets to play">
```

Comment:
`<input>` has no end tag according to html standards, so [must not have an end tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).